lib/components/SimplePage.dart

```diff
@@ -98,7 +98,7 @@ class SimplePage extends StatelessWidget {
     }
 
     return PlatformScaffold(
-        backgroundColor: Theme.of(context).canvasColor,
+        backgroundColor: Theme.of(context).scaffoldBackgroundColor,
```

Comment:
I thought I saw this as part of a different PR, does this just need a rebase?

Reply:
Nope I was thinking of https://github.com/DefinedNet/mobile_nebula/pull/229, which is slightly different.

Reply:
This was #232